### DISCUSSION TITLE
Self hosted access issue

### DISCUSSION THREAD

**Discussion by gratefuluser23** [ghostfolio/ghostfolio#2238] [Q&A]
Have been using Ghostfolio for a while and once DRIP becomes available will be the perfect solution.

The problem I have after suddenly being hospitalised is attempting to login. Is there anyway to change the need to remember a long token to username/password.

**Comment by chart-analyst** [comment]
It's my understanding that once you misplace your token, access to that account is lost.  I would suggest you use a password manager in the future.  I use Bitwarden, which is free for personal use.

**Comment by gratefuluser23** [comment]
Sorry for delay in responding, and I do make use of password managers etc. However, it would be good to have an alternative method for logging in when self-hosting. 

The question/observation/enhancement still stands - is it possible to have username or just password as a way to login when self-hosting?
